src/entity/units/gripthulu.ts

```diff
@@ -28,8 +30,8 @@ const unit: UnitSource = {
   },
   spawnParams: {
     probability: 20,
-    budgetCost: 4,
-    unavailableUntilLevelIndex: 7,
+    budgetCost: 12,
+    unavailableUntilLevelIndex: 6,
```

Comment:
I don't think gripthulus are enough of a threat to take up this much budget

Reply:
12 is on the lower end now: a blood golem/archer only takes 10

We could lower it, this was mainly to distringuish it from filler units.

It also stems from the enemy being.. kind've annoying. Lol. It might not kill you, but it's very intrusive by naturr and thus I thought it should have slightly higher budget than the other guys. I think it was buffed here too.